### PR TITLE
Update common-issues.md

### PR DESCRIPTION
**Pull request by icarusdust**
Fixes https://github.com/codemagic-ci-cd/codemagic-docs/issues/440

### CODE REVIEW

File: content/troubleshooting/common-issues.md [hunk 1]
```diff
@@ -123,3 +123,28 @@ In such cases, you will often see an error message similar to this one:
 **Solution**:
 * Confirm that you have set the correct bundle identifiers for all targets in Xcode.
 * Verify that the bundle identifier set in Xcode matches the bundle identifier set in the Flutter workflow editor under code signing.
+
+
+### Common Podfile issue
```
Comment (helinanever) [inline content/troubleshooting/common-issues.md:128]: The title should be more descriptive about the issue, eg. iOS deployment target issue

File: content/troubleshooting/common-issues.md [hunk 2]
```diff
@@ -123,3 +123,28 @@ In such cases, you will often see an error message similar to this one:
...
+The following issue is usually occurs when it comes to podfile issues:
```
Comment (helinanever) [inline content/troubleshooting/common-issues.md:129]: A common error message with regard to deployment target is the following:

File: content/troubleshooting/common-issues.md [hunk 3]
```diff
@@ -123,3 +123,28 @@ In such cases, you will often see an error message similar to this one:
...
+```
+The iOS deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 14.4.99
+```
+
```
Comment (helinanever) [inline content/troubleshooting/common-issues.md:133]: Add here:
The error usually occurs if the deployment target set in Podfile is lower than required or when the Podfile is missing.

File: content/troubleshooting/common-issues.md [hunk 4]
```diff
@@ -123,3 +123,28 @@ In such cases, you will often see an error message similar to this one:
...
+**Solution**:
+* Confirm if Podfile exists in repo if not, then create Podfile in `ios` directory and insert the following manually in there:
```
Comment (helinanever) [inline content/troubleshooting/common-issues.md:135]:
```suggestion
* Confirm if a Podfile exists in the repository. If not, create a Podfile in the `ios` directory and insert the following manually in there:
```

File: content/troubleshooting/common-issues.md [hunk 5]
```diff
@@ -123,3 +123,28 @@ In such cases, you will often see an error message similar to this one:
...
+# Uncomment the next line to define a global platform for your project
+platform :ios, ’10.0’
+target 'Runner' do
+  # Comment the next line if you don't want to use dynamic frameworks
+  use_frameworks!
+ # Pods for Runner
+end
+```
+
+* Check Podfile, if `platform :ios, ’10.0’` is set to `10.0` or above as required by dependencies
```
Comment (helinanever) [inline content/troubleshooting/common-issues.md:147]:
```suggestion
* Check the Podfile and make sure that `platform :ios, ’10.0’` is set to `10.0` or above as required by dependencies.
```

**Comment by icarusdust** [comment]
changes made

**Review by helinanever — APPROVED** [review]
LGTM

**Pull request merged**